src/helpers/figures/charts/chart_common.ts

```diff
@@ -428,3 +428,21 @@ export function getDefinedAxis(definition: ChartWithAxisDefinition): {
   useLeftAxis ||= !useRightAxis;
   return { useLeftAxis, useRightAxis };
 }
+
+export function computeChartPadding(
+  displayTitle: boolean,
+  displayLegend: boolean
+): {
```

Comment:
```suggestion
export function computeChartPadding({
  displayTitle: boolean,
  displayLegend: boolean
}): {
```
IMO it's easier to read a call like 
```ts
padding: computeChartPadding({
    displayTitle: !!chart.title.text,
    displayLegend: legend.display !== false
}),
``` 
than 
```ts
padding: computeChartPadding(!!chart.title.text, legend.display !== false),
```